src/electionguard_verify/verify.py

```diff
@@ -67,3 +65,17 @@ def verify_decryption(
                     )
 
     return Verification(True, message=None)
+
+
+def verify_aggregation(
+    submitted_ballots: List[SubmittedBallot],
+    tally: CiphertextTally,
+) -> Verification:
+    for ballot in submitted_ballots:
```

Comment:
This check is good for checking the inclusion of the ballots but this is not checking the full aggregation. 
https://github.com/microsoft/electionguard-python/blob/6bf466f3cf8b3f47cd29e1c549f86ad1758b6221/src/electionguard/tally.py#L216-L226

This PR needs to actually check the aggregation. This includes aggregating the submitted ballots into a CiphertextTally and comparing this to the other CiphertextTally's contests.

https://github.com/microsoft/electionguard-python/blob/184c9cf7fdbaaff20a241fa48bcd478a9e751882/src/electionguard/tally.py#L202